web/pages/affiliates/index.md

```diff
@@ -0,0 +1,28 @@
+title: Plasma Packages
+hidetitle: True
+author: Erik T. Everson
+
+The PlasmaPy's charter includes promoting open-source software for all of 
+the plasma community, and not just `plasmapy`.  We feel the entire community 
+benefits when there are multiple efforts trying to build functionality for all 
+aspects of the plasma field.  As such, we want to promote any ongoing effort.  This
+page displays projects that have requested to be shown.  If you want your 
+project include, and increase its visibility, then please submit a request.
+
+<div style="width: 100%">
+    <a href=https://docs.google.com/forms/d/e/1FAIpQLSfz2djEihWkODbkr1ZC5splkX7ZKLAh_Kuy-aXOngTELIm3xQ/viewform?usp=sf_link
+            class="feature-card feature-link btn-plasmapy-bluegreen" 
+            style="width: 200px">
+        <div>Submit Request</div>
+    </a>
+</div>
+<br>
+
+All projects/packages are considered **Community Packages** 
+<span style="vertical-align: 2px">{{% community_shield %}}</span>, but packages designed with
+`plasmapy` interoperability are marked as **Affiliated Packages** 
```

Comment:
```suggestion
PlasmaPy interoperability are marked as **Affiliated Packages** 
```

Reply:
I'm still think package here instead of project because "Affiliated Packages" have interoperability with `plasmapy`, not PlasmaPy.